### DISCUSSION TITLE
How can I achieve this Shader nodes in MALT?

### DISCUSSION THREAD

**Discussion by krigeta** [bnpr/Malt#303] [Q&A] [answered]
Hello, I want to combine this with Line Shader in Malt but don't know how can I do that, may you please help me.

this is the node setup I want to replicate.

![aa](https://user-images.githubusercontent.com/75309361/169979896-6114449b-1021-4d3e-9210-9380db9c0082.png)
?

**Comment by pragma37** [comment, marked as answer]
Here you have it:

![imagen](https://user-images.githubusercontent.com/36610999/170059351-6c089567-b89f-4ad0-83b7-fc69214045b5.png)
